src/main/resources/requests/PublicationRequests.ts

```diff
@@ -6,11 +6,11 @@ import { LogHelper } from '../logging/logHelper';
 const logHelper = new LogHelper();
 
 export class PublicationRequests {
-    public async getIndividualPublicationMetadata(artefactId, userId, admin): Promise<any> {
+    public async getIndividualPublicationMetadata(artefactId, requesterId, admin): Promise<any> {
         try {
             let header;
-            if (userId) {
```

Comment:
Can we temporarily also have the x-user-id here. I'm thinking when we merge, it'd be good to merge frontend first, then merge backend and then after we can remove the old x-user-id

Reply:
This is the same for anywhere where we have replaced the existing ID header that is passed with the new one.  Any ones where we are adding a new header where one didn't exist previously should be fine

Reply:
added x-user-id.